utils/update_checkout/update-checkout-config.json

```diff
@@ -160,7 +160,7 @@
                 "swift-sdk-generator": "main",
                 "wasi-libc": "wasi-sdk-20",
                 "wasmkit": "0.0.3",
-                "curl": "curl-8_5_0",
+                "curl": "curl-8_9_1",
```

Comment:
Please also update the next and rebranch.

Reply:
https://github.com/swiftlang/swift/pull/75868